core/sbt-cloudflow/src/main/scala/cloudflow/sbt/CloudflowFlinkPlugin.scala

```diff
@@ -32,11 +32,12 @@ object CloudflowFlinkPlugin extends AutoPlugin {
   override def requires = CloudflowBasePlugin
 
   override def projectSettings = Seq(
+    cloudflowFlinkBaseImage := None,
     libraryDependencies ++= Vector(
           "com.lightbend.cloudflow" %% "cloudflow-flink"         % BuildInfo.version,
           "com.lightbend.cloudflow" %% "cloudflow-flink-testkit" % BuildInfo.version % "test"
         ),
-    cloudflowDockerParentImage := CloudflowFlinkDockerBaseImage,
+    cloudflowDockerParentImage := cloudflowFlinkBaseImage.value.getOrElse(CloudflowFlinkDockerBaseImage),
```

Comment:
Is `cloudflowDockerParentImage` still used / needed?

Reply:
Good catch. Not used anywhere. Removed.